packages/react-core/src/components/Wizard/WizardToggle.tsx

```diff
@@ -66,23 +69,25 @@ export const WizardToggle: React.FunctionComponent<WizardToggleProps> = ({
 
   return (
     <React.Fragment>
-      <button
-        onClick={() => onNavToggle(!isNavOpen)}
-        className={css(styles.wizardToggle, isNavOpen && 'pf-m-expanded')}
-        aria-label={ariaLabel}
-        aria-expanded={isNavOpen}
-      >
-        <span className={css(styles.wizardToggleList)}>
-          <span className={css(styles.wizardToggleListItem)}>
-            <span className={css(styles.wizardToggleNum)}>{activeStepIndex}</span> {activeStepName}
-            {activeStepSubName && <AngleRightIcon className={css(styles.wizardToggleSeparator)} aria-hidden="true" />}
+      {!hasDrawer && (
```

Comment:
I am not quite understanding why we hare not rendering this if we have a drawer?

Reply:
Good question @tlabaj ! I added this conditional to match the html structure of the core demo: 
<img width="618" alt="Screen Shot 2022-05-06 at 3 43 49 PM" src="https://user-images.githubusercontent.com/32821331/167206782-da9a4367-a46b-49fb-8a9e-234bcfef659f.png">

We don't want to render any of the stuff that comes before [.pf-c-wizard__outer-wrap](https://github.com/patternfly/patternfly-react/pull/7323/files/70ce3b170251b056b04ed7df97ec729e148c9960#diff-038302fe0ddb5ac4ec5b59ca6bf16bd2ef6fa81fcf8bb56572a40d78df73b59eR91)

Reply:
I think the structure may be off.  It looks like the toggle is before the drawer in core.
![image](https://user-images.githubusercontent.com/39532947/167217365-61f069fe-b7c2-4a02-bad1-c050797a18ae.png)

In your pr it is missing al together
![image](https://user-images.githubusercontent.com/39532947/167217607-aadbd853-c294-4aa1-aefc-eb56afca64b8.png)

Reply:
Fixed! I moved `WizardDrawerWrapper` into `WizardToggle` which solved this structural issue - we no longer need that `hasDrawer` condition either! The HTML should match up now 👍 thank you @tlabaj